content/kotlin/concepts/datetime/terms/plus/plus.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: '.plus()'
+Description: 'Adds a datetime object with a specified datetime period.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Data Structures'
+  - 'Time'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.plus()`** function adjusts a DateTime object by adding a specified period of time to it, such as days, hours, or minutes. It can be applied to instances of datetime classes provided by the Kotlin datetime package, such as `Instant`, `LocalDate`, `DateTimePeriod`, and `DatePeriod`.
```

Comment:
```suggestion
The **`.plus()`** function adjusts a DateTime object by adding a specified period of time to it, such as days, hours, or minutes. It can be applied to instances of `datetime` classes provided by the Kotlin `datetime` package, such as `Instant`, `LocalDate`, `DateTimePeriod`, and `DatePeriod`.
```